handbook/engineering/code-intelligence/goals.md

```diff
@@ -14,27 +14,28 @@ Progress on adoption and usage is tracked in our [Looker dashboard](https://sour
 
 **Outcome:**
 
-- 25 popular open source C++ repositories are indexed on Sourcegraph Cloud.
-- 25 popular open source Java repositories are indexed on Sourcegraph Cloud.
+- 30 popular open source C++ repositories are indexed on Sourcegraph Cloud.
+- 30 popular open source Java repositories are indexed on Sourcegraph Cloud.
 - C++ and Java indexers are adopted by at least 3 customers each.
 - Precise code intel reaches [*N<sub>0</sub>*][N0] precise code intel operations a month.
 - CE and Sales have a significant amount of open source repos to showcase and can easily demonstrate the value of setting up precise code intelligence.
 - For languages that are particularly subject to unusual setups and tooling, our indexed open source showcase helps us determine if setup issues are caused by an unusual customer dev environment.
```

Comment:
Drive by comment while this is being updated: I really like how @Joelkw inlined outcomes with the milestones. What do you think about doing the same thing here?

https://about.sourcegraph.com/handbook/engineering/code-insights/goals

Reply:
Thanks the input @nicksnyder ! I'm merging as is for the time being (as I'll be ooo until Monday) but will open a new one next week to add these changes.